projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2109,7 +2110,7 @@ static function parse( $attributes, $content ) {
 			$back_url = remove_query_arg( array( 'contact-form-id', 'contact-form-sent', '_wpnonce' ) );
 
 			$r_success_message =
-				'<h3>' . __( 'Message Sent', 'jetpack' ) .
+				'<h3>' . $form->get_attribute( 'customThankyouHeading' ) .
```

Comment:
Should we add some escaping here?

Reply:
Thank you! I've addressed both issues now.